### PR TITLE
docs: update README

### PR DESCRIPTION
**Pull request by b4nst**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,31 +60,49 @@ nested: configmap: {
...
+You can use it to deploy single Manifest, a bunch of unrelated Manifests or use the [Instance](#instance) concept.
 
-### Attributes
+### Instance
 
+An instance is a group of [Manifests](#manifest) belonging to the same _application_.
+It's a way to group multiple manifests under the same lifecycle.
+If you're used to Helm, you can see a Cuebe Instance as a [Helm Release](https://helm.sh/docs/intro/using_helm/#three-big-concepts).
+
+A Manifest needs the `"cuebe.loft-orbital.com/instance": <instance-name>` label to be a member of the _<instance-name>_ Instance.
```
Comment (astraios) [inline README.md:77]: Do we need to be more specific and state we're talking about a Kubernetes `metadata.labels` ?

**Review by astraios — COMMENTED** [review]
LGTM, I think it might help to have cuebe command example associated with cue examples to illustrate a bit more the relationship between things like tags and -t. or injects or context or -e.... Sorry, that's a lot of `or`

File: README.md (hunk 1 above)
Comment (b4nst) [reply README.md:77]: Yes, very good point

**Comment by b4nst** [comment]
@astraios Yes we definitely lack good examples. Also this PR is about the README.md but there's also auto generated doc for each command when we release. It's then published [here](https://loft-orbital.github.io/cuebe).

**Pull request merged**